src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
Safari does not support any part of SW yet, it simply does not support it. For Chrome on Android we need to ensure that `isSupported()` continues to return `false` when called from the context of a web page, else we will break a lot of people's code in production.